initfiles/componentfiles/configxml/cgencomplist_linux.xml.in

```diff
@@ -51,7 +51,10 @@
   <Component name="backupnode" processName="BackupNode" schema="backupnode.xsd">
     <File name="backupnode_vars.xsl" method="xslt" destName="backupnode.conf"/>
   </Component>
-  @SPARKTHOR_CGENCOMPLIST_LINUX_ENTRY@
+  <Component name="sparkthor" processName="SparkThor" schema="sparkThor.xsd">
```

Comment:
Does this mean sparkthor will always be configurable via configmgr even if the plugin isn't installed?